packages/web/src/store/token-dashboard/signMessage.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  accountSelectors,
+  Chain,
+  tokenDashboardPageActions,
+  tokenDashboardPageSelectors
+} from '@audius/common'
+import { call, put, select } from 'typed-redux-saga'
+
+import { WalletConnection } from './types'
+const { getUserId } = accountSelectors
+const { getConfirmingWallet } = tokenDashboardPageSelectors
+const { updateWalletError } = tokenDashboardPageActions
+
+export function* signMessage(connection: WalletConnection) {
+  const accountUserId = yield* select(getUserId)
+  const message = `AudiusUserID:${accountUserId}`
+  const { wallet } = yield* select(getConfirmingWallet)
+  if (!wallet) return
+
+  if (connection.chain === Chain.Eth) {
```

Comment:
might help for readability if this was a 
```
switch (connection.chain) {
 case Chain.Eth:

 case Chain.Sol
}
```

Reply:
yeah i like that!